base/expvar.go

```diff
@@ -131,6 +131,8 @@ const (
 	StatKeyPullReplicationsCaughtUp         = "num_pull_repl_caught_up"
 	StatKeyRequestChangesCount              = "request_changes_count"
 	StatKeyRequestChangesTime               = "request_changes_time"
+	StatKeyDcpReceivedCount                 = "dcp_received_count"
```

Comment:
There are other DCP stats below these, but it seems wrong to put them in the `StatsCBLReplicationPull` group? Shouldn't they be in `StatsDatabase` or something?

Reply:
It's probably 50-50 - we're only listening to DCP and doing channel caching to support pull replications, but I agree there's a slightly better case for sticking it in Database.  I'll move.